packages/xdl/src/project/Doctor.ts

```diff
@@ -176,6 +176,11 @@ async function _validateExpJsonAsync(
   ProjectUtils.clearNotification(projectRoot, 'doctor-problem-checking-watchman-version');
 
   const sdkVersion = exp.sdkVersion;
+  if (!sdkVersion) {
+    throw new Error(
+      'Config is missing an SDK version. See https://docs.expo.io/bare/installing-updates/'
```

Comment:
i'm not sure i understand this one -- why are we linking to the bare workflow instructions for installing expo-updates if sdk version is missing? is it still not possible to use expo-updates without setting an sdk version?

Reply:
I was assuming that a missing SDK version was evidence that they were trying to do something with a runtime version in bare.

This part of the PR was just me trying to improve validation and not necessary to the rest. So I think I will remove it as I don't have enough context about the issue.